multiOwner.sol

```diff
@@ -112,16 +99,25 @@ contract multiOwner is safeMath {
             return false;
         }
     }
-    /*
-        Privates
-    */
+    /* Constants */
+    function ownersForChange() public constant returns (uint256 owners) {
+        return ownerCount * doConfirmRate / 100;
```

Comment:
Oh, I didn't seen that OpenZeppeling have changed the implementation of the `div` in their `safeMath` library. https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/math/SafeMath.sol#L15-L20

You're right, div is no longer needed and it should be considered deprecated.